src/api/app/controllers/webui/users_controller.rb

```diff
@@ -38,7 +38,9 @@ def show
     # move back to the monday before (make it up to 53 weeks)
     @first_day -= (@first_day.cwday - 1)
 
-    @activity_hash = User::Contributions.new(@displayed_user, @first_day).activity_hash
+    @activity_hash = UserYearlyContribution.new(@displayed_user, @first_day).call
```

Comment:
maybe change the name to something expressing the time range (in the case yearly)

Reply:
Done